ui/src/app/store/utils/slice.ts

```diff
@@ -164,12 +166,16 @@ export const generateSlice = <
   indexKey,
   initialState,
   reducers,
+  extraReducers,
   setErrors,
 }: {
   name: keyof CommonStates;
   indexKey: K;
   initialState?: GenericState<I, E>;
   reducers?: ValidateSliceCaseReducers<GenericState<I, E>, R>;
+  extraReducers?:
```

Comment:
Just double checking - are these only to be used in cases where you want to reduce a state item based on an action meant for another state item?

Reply:
Right! You can use them if you want a reducer to reduce a different action type.